content/learn/sandbox.adoc

```diff
@@ -0,0 +1,64 @@
+---
+menu:
+  learn:
+    parent: Workflow
+title: Validated Patterns - Sandbox Tier
+weight: 42
+aliases: /requirements/community/
+aliases: /requirements/sandbox/
+---
+
+:toc:
+
+= The Validated Patterns 'Sandbox' Tier 
+
+[id="tldr"]
+== tl;dr
+
+The Sandbox tier is an inclusive and low-risk way to become associated with the Validated Patterns initiative.  As long as the patterns framework is your starting point, and you include some minimal documentation, then you are welcome here.
+
+[id="onboarding-existing-implementations"]
+== Onboarding Existing Implementations
+
+TODO: A short note on the value of converting existing implementations
+
+The Validated Patterns team has a preference for empowering others, and not
+taking credit for their work.
+
+Where there is an existing application/demo, there is also a strong preference for the originating team to own any changes that are needed for the implementation to become a Validated Pattern.  Alternatively, if the Validated Patterns team drives the conversion, then in order to prevent confusion and duplicated efforts, we are likely to ask for a commitment to phase out use of the previous implementation for future engagements such as demos, presentations, and workshops.
+
+The goal is to avoid bringing a parallel implementation into existence which divides engineering resources, and creates confusion internally and with customers as the implementations drift apart.
+
+In both scenarios the originating team can choose where to host the primary repository, will be given admin permissions to any fork in https://github.com/validatedpatterns , and will receive on-going assistance from the Validated Patterns team.
+
+[id="requirements"]
+== Requirements
+
+General requirements for all Validated Patterns
+
+[id="must"]
+=== Must
+
+. Sandbox Patterns *MUST* continue to meet the following criteria to remain in the Sandbox tier
+. Sandbox Patterns *MUST* conform to the common technical link:/requirements/implementation/[implementation requirements]
+. Tested Patterns *MUST* be able to be deployed onto a freshly deployed OpenShift cluster without prior modification or tuning
+. Sandbox Patterns *MUST* include a top-level README highlighting the business problem and how the pattern solves it
+. Sandbox Patterns *MUST* include an architecture drawing. The specific tool/format is flexible as long as the meaning is clear.
+. Sandbox Patterns *MUST* undergo an informal technical review by a community leader to ensure that it meets basic reuse standards
```

Comment:
We should probably define "community leader" a bit better here. It's not entirely obvious

Reply:
somewhat deliberate